addon/utils/regexp.js

```diff
@@ -0,0 +1,7 @@
+const versionRegExp = /\d[.]\d[.]\d/;
+const shaRegExp = /[a-z\d]{8}/;
+
+export default {
```

Comment:
Not really, it was moreso if this was ever needed to be exported as an object... but I guess when that case arises, it can be added.